packages/server/src/fhir/repo.ts

```diff
@@ -526,7 +527,18 @@ export class Repository extends BaseRepository implements FhirRepository {
   private validate(resource: Resource): void {
     if (this.context.strictMode) {
       const start = process.hrtime.bigint();
-      validate(resource);
+      try {
+        validate(resource);
+      } catch (err: any) {
+        const invariantErrors = err.outcome.issue.filter((issue: OperationOutcomeIssue) => issue.code === 'invariant');
```

Comment:
In regards to not wanting to block any constraint errors until we have an understanding, we're not throwing if the outcome issues only contain invariant errors